onboarding/schema.py

```diff
@@ -410,3 +424,20 @@ def resolve_onboarding_info(self, info: ResolveInfo) -> Optional[OnboardingInfo]
         if hasattr(user, "onboarding_info"):
             return user.onboarding_info
         return None
+
+    def resolve_onboarding_step_1(self, info: ResolveInfo):
```

Comment:
Are these called anywhere?

Reply:
Oh yeah that's part of the Graphene API, basically if a graphene GraphQL object doesn't have a `resolver` as part of the GraphQL field itself (i.e. the `graphene.Field()`) , then graphene will try looking for a `resolve_<field name>` method on the GraphQL object that has the field on it in order to resolve it.  That's why a bunch of our GraphQL classes have `resolve_<something>` methods that never seem to get explicitly called from anywhere.

Reply:
got it, thanks for explaining!